docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md

```diff
@@ -168,141 +57,49 @@ The following table lists the available Data Asset metrics.
+| **Mean**                                 | For numeric columns, the average value in the column.<br/> This is determined by dividing the sum of all values in the column by the number of values.  |
 | **Median**                                 | For numeric columns, the value in the middle of a data set.<br/> 50% of the data within the Data Asset has a value smaller or equal to the median, and 50% of the data within the Data Asset has a value that is higher or equal to the median.  |
 | **Null %**                                | The percentage of missing values in a column.             |
 
-## Add an Expectation to a Data Asset column
-
-When you create an Expectation after fetching metrics for a Data Asset, the column names and some values are autopopulated for you and this can simplify the creation of new Expectations. Data Asset Metrics can also help you determine what Expectations might be useful and how they should be configured. When you create new Expectations after fetching Data Asset Metrics, you can add them to an existing Expectation Suite, or you can create a new Expectation Suite and add the Expectations to it. 
-
-1. In GX Cloud, click **Data Assets** and then select a Data Asset in the **Data Assets** list.
-
-2. Click the **Overview** tab.
-
-    When you select a new Data Asset, schema data is automatically fetched.
-
-3. Optional. Select one of the following options:
-
-    - Click **Profile Data** if you have not previously returned all available metrics for a Data Asset.
-
-    - Click **Refresh** to refresh the Data Asset metrics.
-
-4. Click **New Expectation**.
-
-5. Select one of the following options:
-
-    - To add an Expectation to a new Expectation Suite, click the **New Suite** tab and then enter a name for the new Expectation Suite.
-
-    - To add an Expectation to an existing Expectation Suite, click the **Existing Suite** tab and then select an existing Expectation Suite.
-
-6. Select an Expectation type. See [Available Expectation types](/cloud/expectations/manage_expectations.md#available-expectation-types).
-
-7. Complete the fields in the **Create Expectation** pane.
-
-8. Click **Save** to add the Expectation, or click **Save & Add More** to add additional Expectations.
-
-
-## Add a Data Asset to an Existing Data Source
-
-Additional Data Assets can only be added to Data Sources created in GX Cloud.
-
-1. In GX Cloud, click **Data Assets** and then select **New Data Asset**.
-
-2. Click the **Existing Data Source** tab and then select a Data Source.
+## Edit a Data Asset
 
-3. Click **Add Data Asset**.
+You can only edit the settings of Data Assets created in GX Cloud.
 
-4. Select tables to import as Data Assets:
+1. In GX Cloud, click **Data Assets** and in the Data Assets list click **Edit Data Asset** for the Data Asset you want to edit.
```

Comment:
```suggestion
1. In GX Cloud, click **Data Assets** and in the Data Assets list, click **Edit Data Asset** for the Data Asset you want to edit.
```

Reply:
What is your docs SME recommendation for instructions that direct the user to click a button with an image vs. a button with text? Since the edit button is a small pencil icon and the text only appears when the user mouses over it, do you think users might get confused on this step?

Reply:
ideally we should follow [google developer docs style guide for this](https://developers.google.com/style/ui-elements#buttons)

> If the button includes an icon, write the name of the button as shown in the tooltip, and add the button icon immediately before the name.

<img width="490" alt="Screenshot 2024-11-22 at 2 53 36 PM" src="https://github.com/user-attachments/assets/334cafe6-b158-42b3-8e82-5144a9c85881">

I'll file a separate issue for doing this throughout the cloud docs

Reply:
I didn't commit your suggestion directly because I decided to break this long-winded step into multiple steps. But, I did add the recommended comma after breaking this up.